src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -999,14 +980,13 @@ public EntityDataKeySearchResult executeSearch(
         for ( int i = 0; i < searchConstraints.getEntitySetIds().length; i++ ) {
             UUID entitySetId = searchConstraints.getEntitySetIds()[ i ];
 
-            Optional<Set<UUID>> normalEntitySets = Optional.ofNullable( linkingEntitySets.get( entitySetId ) );
+            Set<UUID> normalEntitySets = linkingEntitySets.getOrDefault(
+                    entitySetId,  DelegatedUUIDSet.wrap(  ImmutableSet.of( entitySetId ) ) );
```

Comment:
why do you need a DelegatedUUIDSet here?

Reply:
Because the return value of linkingEntitySets.getOrDefault "get" case is a  DelegatedUUIDSet

Reply:
oh lol makes sense